### DISCUSSION TITLE
Adding Playbook if this was not installed during setup phase

### DISCUSSION THREAD

**Discussion by mciantar** [Security-Onion-Solutions/securityonion#3744] [Unsupported Versions] [answered]
Is there the possibility of installing Playbook if this was not installed during the initial setup phase please?

Thanks!

**Comment by defensivedepth** [comment, marked as answer]
This is not fully tested, but it should work - run the following:

`sudo so-playbook-reset`

It will take a while.

**Comment by RagnarSTS** [comment]
I also did not install playbook when I did an install on an airgap system.  I performed the so-playbook-reset as suggested and it did complete successfully.  It did not however end up loading in even a basic layout on the attack navigator.  The navigator page on the airgap system is completely blank.  
I'm guessing something additional needs to be downloaded from https://github.com/SigmaHQ/sigma/tree/master/rules  as explained on the soctopus SO page; but not quite sure how to perform it manually.  
Anybody have any other suggestions for fixing up soctopus playbook and getting a basic navigator page?

**Reply by dougburks** [reply]
@RagnarSTS From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and simply provide a hyperlink to the related discussion.
